requirements.txt

```diff
@@ -8,3 +8,9 @@ scikit-learn
 tabulate
 flair
 requests
+scipy==1.10.1
```

Comment:
Do we need to pin the exact version of all of these libraries? Couldn't we just specify the minimum and/or maximum versions that work with our code? I'm afraid that pinning very common libraries like `scipy` and `matplotlib` will cause conflicts with other libraries.

Reply:
In PR #37, I was able to fix the errors by using just:
```
scipy<1.13
matplotlib<3.9
```
The root cause of the errors is in the packages PolyFuzz and gensim, which are transitive dependencies.

Reply:
Yes, I agree, it's better to have flexible versions. I relaxed the versions of the other dependencies.